### PR TITLE
[RELENG-7502-aws-tag] Upgrading boto3 version

### PR DESCRIPTION
**Pull request by gaspardmoindrot**

### CODE REVIEW

**Review by nootal — CHANGES_REQUESTED** [review]
Make sure your commit message contains the reference to the ticket, otherwise this information will be lost once the branch is merged.

File: config_example.yml [hunk 1]
```diff
@@ -37,7 +35,23 @@ python_config: 'settings.settings_local'
 #      tags:
 #        - centos7
 #        - small
-runner_pool: []
+runner_pool:
```
Comment (Abubakarr99) [inline config_example.yml:38]: Is it recommended to hard cord this values in a public repository ? Can't the values be updated in devinfra for example the runner pool and over here we leave it empty. I'm talking about values such as the security_group_ids, subnet_id. Or maybe I'm missing something

Comment (gaspardmoindrot) [reply config_example.yml]: This is a mistake, I did not want to push this file! I just deleted it again

**Review by Abubakarr99 — APPROVED** [review]
LGTM

**Pull request merged**